collectors/timex.plugin/README.md

```diff
@@ -0,0 +1,25 @@
+<!--
+title: "timex.plugin"
+description: "Monitor the system clock synchronization state."
+custom_edit_url: https://github.com/netdata/netdata/edit/master/collectors/timex.plugin/README.md
+-->
+
+# timex.plugin
+
+This plugin monitors the system clock synchronization state, under Linux.
+
+Two charts are available:
+
+-   Clock Synchronization State
+-   Time Offset
```

Comment:
```suggestion
-   System Clock Synchronization State
-   Time Offset Between Local System and Reference Clock
```

Reply:
I’d go a step further and make it ‘computed time offset’. Sometimes that computation will be _very_ accurate (for example on a system syncing off of a number of high quality stratum 1 or 2 NTP servers), sometimes it will be a rough estimate at best (for example, if syncing from a system that is not, itself, tied to a specific reference clock).

Reply:
Updated.